README.md

```diff
@@ -307,6 +307,85 @@ When a test fails, the test suite will copy the pod logs in order to troubleshoo
 ./mvnw clean verify -Dts.instance-logs=../custom/path
 ```
 
+### Using Keycloak Test Resource for Quarkus Tests
+
+In order to use a Keycloak test resource for running the Quarkus Tests, we need to use the `KeycloakQuarkusTestResource` test resource in our Quarkus test:
+
+```java
+@QuarkusTest
+@QuarkusTestResource(KeycloakQuarkusTestResource.class)
+public class MyQuarkusTest {
+
+    @Test
+    public void test_XXX() throws Exception {
+        // ...
+    }
+    
+}
+```
+
+The keycloak test resource is deployed by the [Testcontainers](https://www.testcontainers.org/) framework, so we'll need to add this dependency into the `pom.xml`:
+
+```xml
+<dependency>
+    <groupId>org.testcontainers</groupId>
+    <artifactId>testcontainers</artifactId>
+    <scope>test</scope>
+</dependency>
+```
+
+And also, we'll need to configure the `keycloak-realm.json` in the test classpath of our module. See the next realm configuration as an example:
+
+```json
```

Comment:
Maybe, adding the realm configuration is too much. What do you think?

Reply:
Maybe we should create a security Readme, for the details. And leave this readme as an overview. (link this readme with security readme by a "moreDeatils link")